README.md

```diff
@@ -115,174 +101,60 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+GETTING STARTED 
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
+A new develpoer can use this project to learn how to build a portfolio.
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
-
+   git clone git@github.com:david30-maker/Mobile-first-project.git
+Prerequisites
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
+NodeJS
+Browser
+Setup
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
-### Install
-
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
+  git clone git@github.com:david30-maker/Mobile-first-project.git
+  cd Mobile-first-project
 
-```sh
-  rails server
-```
---->
+Install:
+Install project dependencies in this project with:
 
-### Run tests
+  npm install
 
-To run tests, run the following command:
+Usage:
+To run the project, open the project with live server and visit http://localhost:5500
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
-```
- -->
+Open in browser
+http://localhost:5500
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
+## 👥 Author
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **David Igbo**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@david30-maker](https://github.com/githubhandle)
+- Twitter: [@davidigbo](https://twitter.com/twitterhandle)
+- LinkedIn: [linkedin.com/in/dvid-igbo-464196184/]](https://linkedin.com/in/linkedinhandle)
 
 👤 **Author2**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [OPTIONAL] I think removing this will look better as there is no second author in this repo.